org.jacoco.core/src/org/jacoco/core/internal/analysis/MethodAnalyzer.java

```diff
@@ -162,6 +215,12 @@ public void visitInvokeDynamicInsn(final String name, final String desc,
 	@Override
 	public void visitJumpInsn(final int opcode, final Label label) {
 		visitInsn();
+
+		if (remappedJumps.containsKey(currentNode)) {
```

Comment:
note for myself: should be covered by a synthetic unit test, because as of today CI build always uses javac, while this is used only for ECJ